src/mpi/init/init_impl.c

```diff
@@ -107,17 +109,118 @@ const char **MPIR_pset_list = default_pset_list;
 
 /* Implementations */
 
+/**
+ * @brief   Convert a thread level string into an int representation
+ *
+ * @param   value_str   Thread level string (input)
+ * @param   value_i     Thread level int (output)
+ *
+ * @return  int         MPI_SUCCESS or MPI_ERR_OTHER on error (NULL pointers, unknown thread level)
+ */
```

Comment:
Let's skip the doc comments for static internal functions. I don't think it provides too much value anyway even for Doxygen documents since these are internal. Besides, we don't do that for most of our internal functions, so I would rather keep it consistent.

Reply:
Ok. Removed the doc comments.